CHANGELOG.md

```diff
@@ -1,32 +1,33 @@
-## v.0.1.11:
-* removed support for kibana2 ( use 0.1.10 releases )
-* added serverspec tests
+## 0.1.11:
+* removed support for kibana2 ( use 0.1.10 release if Kibana2 is needed )
+* added serverspec tests. Submitted by Dmitry and Scott (cyberflow and drenalin23).
+* added chefspec unit tests. Submitted by Scott (drenalin23).
 
-## v0.1.10:
```

Comment:
This is presumably a change of tooling during past releases (some tools prepend a `v` to the tagname, some don't. Nice catch.